### ISSUE TITLE
Question : does hid-remapper clone all descriptors of the device ?

### ISSUE THREAD

**Issue by xf6jx5fq** [jfedor2/hid-remapper#80]
Hi,

Just wanted to know if the descriptors are reused and hid remapper device looks like if the device connected to it is actually connected to the computer ?

Thank you.

**Comment by jfedor2** [comment]
No, it has a fixed descriptor that it presents to the computer, regardless of what devices are connected to it.
